examples/SETUP.md

```
@@ -0,0 +1,227 @@
+
+```bash
+export PLANE_ACCESS_TOKEN="your-access-token-here"
+```
+
+**Important:** Only use ONE authentication method - either API key or access token, not both.
+
+### Workspace Slug
+
+The workspace slug is the identifier for your workspace. You can find it in the URL when you're viewing your workspace:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)